releases/release-7.2.0.md

```diff
@@ -0,0 +1,341 @@
+---
+title: TiDB 7.2.0 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 7.2.0.
+---
+
+# TiDB 7.2.0 Release Notes
+
+Release date: xx xx, 2023
+
+TiDB version: 7.2.0
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.2/quick-start-with-tidb) | [Installation packages](https://www.pingcap.com/download/?version=v7.2.0#version-list)
+
+7.2.0 introduces the following key features and improvements:
+
+<table>
```

Comment:
@qiancai @Oreoxmt Not sure the best way to include links to docs so will need your guidance there. Each with a respective documentation page should have a link in the feature name. I see two different ways of hyperlinking in the docs: 1) with the actual URL of the docs page and 2) a relative reference to a docs page like `[Documentation](<path>)`

Reply:
@samhld It depends on where we use these links. 
- For links in Markdown, the syntax is [Documentation](<relative .md path>).
- For links in HTML (like this feature table HTML), because the relative path does not work, we need to use the actual URL.